docs/src/Table.doc.js

```diff
@@ -259,6 +259,362 @@ card(
   />,
 );
 
+card(
+  <Example
+    id="stickyColumn"
+    name="Example: Sticky Column"
```

Comment:
I';d reduce the container width to make the table scroll horizontally. Otherwise the window must be resized down

![image](https://user-images.githubusercontent.com/10593890/113352150-e4f23780-9309-11eb-809e-566997b617cf.png)